chapter_CI_hypothesis/CI_and_hypothesis_test.Rmd

```diff
@@ -0,0 +1,1088 @@
+---
+title: "ScPoEconometrics"
+subtitle: "Confidence Intervals and Hypothesis Testing"
+author: "Florian Oswald, Gustave Kenedi and Pierre Villedieu"
+date: "SciencesPo Paris </br> `r Sys.Date()`"
+output:
+  xaringan::moon_reader:
+    lib_dir: libs
+    css: [default, "../css/scpo.css", "../css/scpo-fonts.css"]
+    nature:
+      beforeInit: ["../js/ru_xaringan.js"]
+      highlightStyle: github
+      highlightLines: true
+      countIncrementalSlides: false
+      ratio: "16:9"
+    includes:
+      in_header: "../libs/partials/header.html"
+---
+
+layout: true
+
+<div class="my-footer"><img src="../img/logo/ScPo-shield.png" style="height: 60px;"/></div> 
+
+---
+
+```{r setup, include=FALSE,warning=FALSE,message=FALSE}
+options(htmltools.dir.version = FALSE)
+knitr::opts_chunk$set(
+  message = FALSE,
+  warning = FALSE,
+  dev = "svg",
+  cache = TRUE,
+  fig.align = "center"
+  #fig.width = 11,
+  #fig.height = 5
+)
+
+# define vars
+om = par("mar")
+lowtop = c(om[1],om[2],0.1,om[4])
+
+library(magrittr)
+library(moderndive)
+library(dplyr)
+library(tidyverse)
+library(infer)
+
+overwrite = FALSE
+
+```
+
+# Recap from last week
+
+* Sampling: ***sampling variation*** and ***sampling distributions***
+
+* population, population parameter, sample statistic or point estimate, random sampling
+
+* ***Unbiased estimator***: $\mathop{\mathbb{E}}[\hat{p}] = p$
+
+* ***Central Limit Theorem***: as sample size increases, sampling distribution is more ***normally distributed***, centered around the ***population parameter***, and has a smaller ***standard error***
+
+--
+
+## Today: Deeper dive into ***statistical inference*** <sup>1</sup>
+
+.footnote[
+[1]: This lecture is based on the wonderful [confidence interval](https://moderndive.com/8-confidence-intervals.html) and [hypothesis testing](https://moderndive.com/9-hypothesis-testing.html) chapters of [ModernDive](https://moderndive.com/)]
```

Comment:
footnote overwrites last bullet points for me

Reply:
hmm on my end it doesn't. does it overwrite it completely?

Reply:
yes, cannot read it. and I have got a *really* big screen. I'd go for removing some bullets to be on the safe side.